platform/public/common/utility.json

```diff
@@ -0,0 +1,47 @@
+{
+    "tool": 
+        {
+            "id": "utility",
+    
+            "name": "Common Utilities",
+    
+            "version": "0.0.1",
+            
+            "author": "MDENet",
+            
+            "homepage": "",
+    
+            "functions": [    
+            ],
+    
+    
+    
+            "panelDefs": [
+    
+                {
+    
+                    "id": "console",
+    
+                    "name": "console",
+    
+                    "panelclass": "ConsolePanel",
+    
+                    "icon": "images/console.gif"
```

Comment:
Where does this image file live?

Reply:
@szschaler  when direct the path used it is relative to the playground base url so it'll be http://127.0.0.1/8080/images/console.gif.

The other way of referencing icons is by using the css id which are loaded relative to the location of the css e.g. the tool static urls.

Reply:
Thanks for clarifying.